### ISSUE TITLE
Cannot connect to daemon at  ws://localhost:55400

### ISSUE THREAD

**Issue by hhenrry501** [Chia-Network/chia-blockchain#8532]
I am on Windows 10, and running chia CLI on this main machine with 4 harvesters. I typed "chia start all" command to start the farming process.
Today I got this "cannot connect to daemon" warning a lot from debug.log 
But it seems this warning doesn't affect my farming status..as it says synced/farming when I type "chia plotnft show/chia farm summary"..
Wondering what might be the causes for this issue?  and can I just ignore this message? Thanks!
![20210919173215](https://user-images.githubusercontent.com/83000063/133922580-db5d3511-a982-4ef0-9e18-8a565e0846b5.png)

**Comment by hhenrry501** [comment]
Seems to be fixed when I just start chia with "chia start full node" command...

**Comment by loppefaaret** [comment]
the `daemon` service might have crashed, or otherwise closed down.
if you are able to find anything on that, please add it

**Comment by hhenrry501** [comment]
> the `daemon` service might have crashed, or otherwise closed down.
> if you are able to find anything on that, please add it

I think I got this one worked out by restart deamon and chia. 
But, I got a new problem. I have two keys here on same machine, one of them is synced to date, no problem, but the other one was a new wallet and I got this error when I was trying to sync it on Windows GUI version 1.2.5. 
Wallet Status from top right is Syncing (0) but the I got error in debug log: 

2021-09-20T23:13:37.100 wallet chia.wallet.wallet_node    : ERROR    Error while trying to fetch from peer:Peer returned no response Traceback (most recent call last):
  File "chia\wallet\wallet_node.py", line 555, in batch_sync_to_peak
  File "chia\wallet\wallet_node.py", line 687, in fetch_blocks_and_validate
ValueError: Peer returned no response

2021-09-20T23:13:37.103 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 0-32. Traceback (most recent call last):
  File "chia\wallet\wallet_node.py", line 608, in sync_job
  File "chia\wallet\wallet_node.py", line 668, in _sync
  File "chia\wallet\wallet_node.py", line 565, in batch_sync_to_peak
RuntimeError: Was not able to add blocks 0-32

Is there another way to sync this wallet?
